src/plugins/rollbar.js

```diff
@@ -0,0 +1,31 @@
+import { version } from '_/package.json';
+
+import Rollbar from 'rollbar';
+
+const config = {
+  accessToken: import.meta.env.VITE_ROLLBAR_ACCESS_TOKEN,
+  captureUncaught: true,
+  captureUnhandledRejections: true,
+  payload: {
+    environment: import.meta.env.PROD ? 'production' : 'development',
+    client: {
+      javascript: {
+        code_version: version,
+      }
+    }
+  },
+};
+
+const rollbar = new Rollbar(config);
+
+export default {
```

Comment:
@itsalaidbacklife there's a lot more we can do with the settings here, https://docs.rollbar.com/docs/rollbarjs-configuration-reference, so I think we should look in to this before merging to make sure we have all the info we need, only track errors, etc.